client/src/components/Reviews/Reviews.jsx

```diff
@@ -4,24 +4,66 @@ import ReviewMapper from './ReviewMapper.jsx';
 import MetaReview from './MetaReview.jsx';
 import Form from './Form.jsx';
 
+import { Container } from '../styles/Container.styled.js';
+import { Button } from '../styles/Button.styled.js';
+
+
+
 //hooks only in stateless components aka not classes
 //reviews
 
-const Reviews = ({ currentProductId, Container, Button, StarReview, currentProductRating }) => {
-  let startPoint = 2;
-  const [metaReview, setMetaReview] = useState([]);
+const Reviews = ({ currentProductId, currentProductRating, totalReviews }) => {
+  const [startPoint, setStart] = useState(0);
+
+  //reviewList states and hooks
   const [reviewList, setReviewList] = useState([]);
+  const [reviewPage, setPage] = useState(1);
+  const [catchUpPage, incPage] = useState(1);
+  const [moreResults, noMoreresults] = useState(true);
+  const [sort, toggleSort] = useState('newest');
+
+  //metaReview states and hooks
+  const [metaReview, setMetaReview] = useState({});
   const [form, formClick] = useState(false);
 
   let getTwoReviews = (id) => {
-    axios.get('/reviews', { params: { id: id } })
-      .then(results => { setReviewList(results.data.results.splice(0, startPoint)); startPoint += 2; })
+    axios.get('/reviews', { params: { id: id, page: reviewPage, sort: sort } })
```

Comment:
As a reminder from standup, you can set count = any number of reviews you want to get, so you don't have to deal with different pages.